providers/aws/resources/aws.lr

```diff
@@ -852,6 +852,8 @@ private aws.iam.role @defaults("arn name") {
   tags map[string]string
   // Time when the role was created
   createDate time
+  // The policy document that grants an entity permission to assume the role
+  assumeRolePolicyDocument dict
```

Comment:
i wonder if we shouldn't make this a real resource object?  is there a specific test you're targeting with this resource? what would the query look like with this as a dict vs a defined resource?

Reply:
Thanks for your feedback. So far I wrote like following:

![Screenshot from 2024-02-29 18-39-45](https://github.com/mondoohq/cnquery/assets/56231339/3ba53ab9-5d7c-498f-a390-377703006769)

Reply:
ah, that makes sense. prob better to just leave as a dict